qcelemental/util/autodocs.py

```diff
@@ -44,6 +44,12 @@ def parse_type_str(prop) -> str:
         fields.SHAPE_LIST: "List",
         fields.SHAPE_SINGLETON: "Union",
     }  # yapf: disable
+    try:
```

Comment:
Ah, I was waiting for the 1.8 regressions. I pin pydantic minor these days until 1.x.1 is out, its getting to be a pretty complex system and regardless of their expansive test suite, items filter through.

Reply:
the big v1.8 one was intended (see breaking: https://github.com/samuelcolvin/pydantic/releases/tag/v1.8). this little typing one was a surprise. Is it supposed to be `Dict` or `Dict[{}]`?